Error: Triple:148274
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/13.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/13.rdf> .'

Error: Triple:270193
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73.rdf> .'

Error: Triple:302572
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/76T.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/76T.rdf> .'

Error: Triple:312339
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at * <http://data.southampton.ac.uk/syllabus/room-feature/006701027/RSC-_SEATS_FIXED_TIERED.rdf?_metadata=all,views,formats,execution,bindings> .):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Froomx2Dfeaturex2F006701027x2FRSCx2Dx5FSEATSx5FFIXEDx5FTIEREDx2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxex78ecution <http://purl.org/linked-data/api/vocab#termBinding> _:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Froomx2Dfeaturex2F006701027x2FRSCx2Dx5FSEATSx5FFIXEDx5FTIEREDx2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://data.southampton.ac.uk/syllabus/room-feature/006701027/RSC-_SEATS_FIXED_TIERED.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:312571
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Predicate must be uriref):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Froomx2Dfeaturex2F006701027x2FRSCx2Dx5FSEATSx5FFIXEDx5FTIEREDx2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://purl.org/linked-data/api/vocab#label> "*" <http://data.southampton.ac.uk/syllabus/room-feature/006701027/RSC-_SEATS_FIXED_TIERED.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:312571
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Predicate must be uriref):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Froomx2Dfeaturex2F006701027x2FRSCx2Dx5FSEATSx5FFIXEDx5FTIEREDx2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://purl.org/linked-data/api/vocab#property> <http://purl.org/linked-data/api/vocab#allProperties> <http://data.southampton.ac.uk/syllabus/room-feature/006701027/RSC-_SEATS_FIXED_TIERED.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:450519
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/F5.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/F5.rdf> .'

Error: Triple:508218
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/30.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/30.rdf> .'

Error: Triple:595897
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73B4.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73B4.rdf> .'

Error: Triple:719811
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73R.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73R.rdf> .'

Error: Triple:736839
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/72A1-B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/72A1-B.rdf> .'

Error: Triple:930408
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/85.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/85.rdf> .'

Error: Triple:1090628
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-33.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-33.rdf> .'

Error: Triple:1206570
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/35.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/35.rdf> .'

Error: Triple:1264020
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73G.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73G.rdf> .'

Error: Triple:1280780
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at * <http://data.southampton.ac.uk/syllabus/location/ECS_Lab2.rdf?_metadata=all,views,formats,execution,bindings> .):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Flocationx2FECSx5FLab2x2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxex78ecution <http://purl.org/linked-data/api/vocab#termBinding> _:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Flocationx2FECSx5FLab2x2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://data.southampton.ac.uk/syllabus/location/ECS_Lab2.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:1281401
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Predicate must be uriref):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Flocationx2FECSx5FLab2x2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://purl.org/linked-data/api/vocab#label> "*" <http://data.southampton.ac.uk/syllabus/location/ECS_Lab2.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:1281519
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Predicate must be uriref):
u'_:httpx3Ax2Fx2Fdatax2Esouthamptonx2Eacx2Eukx2Fsyllabusx2Flocationx2FECSx5FLab2x2Erdfx3Fx5Fmetadatax3Dallx2Cviewsx2Cformatsx2Cex78ecutionx2Cbindingsxxtermx5F* <http://purl.org/linked-data/api/vocab#property> <http://purl.org/linked-data/api/vocab#allProperties> <http://data.southampton.ac.uk/syllabus/location/ECS_Lab2.rdf?_metadata=all,views,formats,execution,bindings> .'

Error: Triple:1288581
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/165.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/165.rdf> .'

Error: Triple:1321377
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/org/HN.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/org/HN.rdf> .'

Error: Triple:1422961
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-14.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-14.rdf> .'

Error: Triple:1455850
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/72CC.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/72CC.rdf> .'

Error: Triple:1464898
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73X-T.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73X-T.rdf> .'

Error: Triple:1695293
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/72B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/72B.rdf> .'

Error: Triple:1716995
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1104.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1104.rdf> .'

Error: Triple:1730043
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1505.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1505.rdf> .'

Error: Triple:1824154
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/73B.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/73B.rdf> .'

Error: Triple:1853378
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-32.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-32.rdf> .'

Error: Triple:1898034
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/9580.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/9580.rdf> .'

Error: Triple:1941349
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1610.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1610.rdf> .'

Error: Triple:1975065
Traceback (most recent call last):
  File "getResources.py", line 165, in processTriple
    g.parse(data=line,format="nquads")
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1492, in parse
    context.parse(source, publicID=publicID, format=format, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/graph.py", line 1039, in parse
    parser.parse(source, self, **args)
  File "/usr/local/lib/python2.7/dist-packages/rdflib-4.2.2-py2.7.egg/rdflib/plugins/parsers/nquads.py", line 66, in parse
    raise ParseError("Invalid line (%s):\n%r" % (msg, __line))
ParseError: Invalid line (Failed to eat [ \t]*\.[ \t]*(#.*)? at @EN <http://data.southampton.ac.uk/building/1730-09.rdf> .):
u'<http://id.southampton.ac.uk/> <http://www.w3.org/2000/01/rdf-schema#label> "University of Southampton"@EN <http://data.southampton.ac.uk/building/1730-09.rdf> .'

